.drone.yml

```diff
@@ -307,95 +319,143 @@ trigger:
-      platforms:
-        - linux/amd64
-        - linux/arm64
-    when:
-      event:
-        - tag
-  - name: push-agent-manifest
-    image: plugins/manifest:1.0.2
-    settings:
-      username:
-        from_secret: docker_username
-      password:
-        from_secret: docker_password
-      target: "rancher/rancher-agent:${DRONE_TAG}"
-      template: "rancher/rancher-agent:${DRONE_TAG}-ARCH"
-      platforms:
-        - linux/amd64
-        - linux/arm64
-    when:
-      event:
-        - tag
+- name: push-head-manifest
+  image: plugins/manifest:1.1.0
+  settings:
+    ignore_missing: true
+    password:
+      from_secret: docker_password
+    spec: manifest.tmpl
+    username:
+      from_secret: docker_username
+  when:
+    ref:
+      include:
+      - "refs/heads/master"
+      - "refs/heads/release/v*"
+    event:
+    - push
+
+- name: push-head-agent-manifest
+  image: plugins/manifest:1.1.0
+  settings:
+    ignore_missing: true
+    password:
+      from_secret: docker_password
+    spec: manifest-agent.tmpl
+    username:
+      from_secret: docker_username
+  when:
+    ref:
+      include:
+      - "refs/heads/master"
+      - "refs/heads/release/v*"
+    event:
+    - push
+
+- name: push-manifest
+  image: plugins/manifest:1.1.0
+  settings:
+    ignore_missing: true
+    password:
+      from_secret: docker_password
+    spec: manifest.tmpl
+    username:
+      from_secret: docker_username
+  when:
+    event:
+    - tag
+
+- name: push-agent-manifest
+  image: plugins/manifest:1.1.0
+  settings:
+    ignore_missing: true
+    password:
+      from_secret: docker_password
+    spec: manifest-agent.tmpl
+    username:
+      from_secret: docker_username
+  when:
+    event:
+    - tag
+
+- name: build-chart
+  image: rancher/dapper:v0.4.1
+  commands:
+  - dapper chart/ci
+  privileged: true
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+  when:
+    event:
+    - push
+    - tag
+
+- name: chart-publish
+  image: plugins/gcs
+  settings:
+    acl:
+    - allUsers:READER
+    cache_control: "public,no-cache,proxy-revalidate"
+    source: bin/chart
+    target: releases.rancher.com/server-charts
+    token:
+      from_secret: google_auth_key
+  when:
+    event:
+    - tag
+
+- name: slack_notify
+  image: plugins/slack
+  settings:
+    template: "Build {{build.link}} failed to push manifests.\n"
+    username: Drone_Publish
+    webhook:
+      from_secret: slack_webhook
+  when:
+    event:
+      exclude:
+      - pull_request
+    instance:
+    - drone-publish.rancher.io
+    status:
+    - failure
+
+volumes:
+- name: docker
+  host:
+    path: /var/run/docker.sock
 
 trigger:
   event:
     exclude:
     - promote
 
 depends_on:
-- default-amd64
-- default-arm64
+- default-linux-amd64
+- default-linux-arm64
+
 ---
 kind: pipeline
 name: publish
 
 platform:
   os: linux
-  arch: amd64 
+  arch: amd64
 
 steps:
 - name: chart-promote
-  pull: default
   image: rancher/dapper:v0.4.1
```

Comment:
Why did we remove this?

Reply:
The drone team indicated that it doesn't do anything. `pull: Always` is documented as an option, but setting it to `default` is not. As far as I can tell it is a holdover from the 0.8 syntax migration.